Source/swiftlint/Commands/LintCommand.swift

```diff
@@ -55,7 +55,10 @@ struct LintCommand: CommandType {
             violations += currentViolations
             reporter.reportViolations(currentViolations, realtimeCondition: true)
         }.flatMap { files in
-            reporter.reportViolations(violations, realtimeCondition: false)
+            if isWarningThresholdBroken(configuration, violations: violations) {
+                violations.append(createThresholdViolation(configuration.warningThreshold!))
+            }
+            reporter.reportViolations(violations, realtimeCondition: true)
```

Comment:
Done. With the tests I cant seem to find anywhere there are tests for the LintCommand.

Would you like me to create a test class in this swiftlint group or make one in the SwiftLintFrameworkTests